tests/cpu/test_kalman_fitter_wire_chamber.cpp

```diff
@@ -149,6 +149,15 @@ TEST_P(KalmanFittingWireChamberTests, Run) {
             const auto& fit_res = track_states[i_trk].header;
             const auto& track_states_per_track = track_states[i_trk].items;
 
+            const auto& seed = track_candidates[i_trk].header;
+            const auto& cands = track_candidates[i_trk].items;
+
+            if (cands.size() < 20) {
+
+                std::cout << "Track momentum (p, pT): " << seed.p() << " "
+                          << seed.pT() << " " << seed.qop() << " " << seed.phi()
+                          << " " << seed.theta() << std::endl;
+            }
```

Comment:
Was this meant to stay in, or was this just a debug measure?

Reply:
Good catch. I removed it